src/eq_schema/Block.js

```diff
@@ -3,12 +3,18 @@ const RoutingRule = require("./RoutingRule");
 const RoutingDestination = require("./RoutingDestination");
 const { get, isNil, remove, isEmpty } = require("lodash");
 const { getInnerHTML } = require("../utils/HTMLUtils");
+const { flow, getOr, last, map, some } = require("lodash/fp");
 
 const pageTypeMappings = {
   QuestionPage: "Question",
   InterstitialPage: "Interstitial"
 };
 
+const getLastPage = flow(getOr([], "pages"), last);
+
+const isLastPageInSection = (page, ctx) =>
+  flow(getOr([], "sections"), map(getLastPage), some({ id: page.id }))(ctx);
```

Comment:
If you upgrade the `prettier` dependency, it will format this in a much easier to read way. Latest prettier gives special treatment to things like `flow`, `compose` etc. See https://prettier.io/blog/2018/05/27/1.13.0.html#format-functional-composition-more-nicely-4431-by-suchipi